app/views/price_policies/instrument/_minimum_calculation.html.haml

```diff
@@ -0,0 +1,9 @@
+- rate = price_policy.minimum_cost
```

Comment:
What if you pulled these out of this partial and passed them in as local variables. Then /instrument/_rate_calculation.html.haml could share this same partial. Usually I don't like this many assignments within a partial.